src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -551,12 +551,38 @@ public function validateDateFormat($attribute, $value, $parameters)
             return false;
         }
 
-        foreach ($parameters as $format) {
+        $formats = array_filter($parameters, fn (int|string $v) => $v !== 'loose');
+        $loose = $formats !== $parameters;
```

Comment:
Wouldn't it be better to use the "in_array" function?

Reply:
It is an ingenious way of eliminating the word 'loose' in advance, so that the foreach loop does not need to check whether the word is loose or not.
In your proposal, in_array('loose', $parameters, true) is expected, but I think the current implementation is better in that you do not have to write "loose" twice.